### PR TITLE
OC-152 Add paragraph to how to start contributing.

### PR DESCRIPTION
**Pull request by Sander3003**

### CODE REVIEW

**Comment by Sander3003** [comment]
@PieterZeeuw can you check my pull-request?

File: Opensourcecommunity/Getstarted.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This is guidance to start contributing to the open smart grid platform project:
```
Comment (pczeeuw) [inline Opensourcecommunity/Getstarted.md]: Maybe change 'This is guidance' into 'This is a guide to start contributing ...' 

or 'Please follow the steps below to start contributing..'

**Comment by pczeeuw** [comment]
Everything looks good, just one small thing about the Guidance part.

**Comment by pczeeuw** [comment]
@Sander3003 
Is the branch still building? It looks like you commited 45 mins ago, but the jenkins check is still pending..

**Comment by Sander3003** [comment]
Building the OSGP_Integration-Tests_development took 59 minutes. This pull request had to wait.

**Pull request merged**